docker/dev_env/run.sh

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env bash
+
+set -e
+
+volume_name="openverse-dev-env"
+
+if ! docker volume inspect openverse-dev-env 2>/dev/null 1>/dev/null; then
```

Comment:
I always get confused, is `[ ... ]` not needed here? Also `&>`.

```suggestion
if ! docker volume inspect openverse-dev-env &>/dev/null; then
```

Reply:
The square brackets aren't needed when testing the return code of the command, which is what we're doing here. `[` is actually an alias for `test` and `]` is just a conventional way to close it (but actually `;` does the closing if you put `then` on the same line). `[[` is a special bash extension separate from `[`.

If we used `[..]` it would run the contents through `test`, which we don't want here. Instead we just use `!` to assert whether the exit code is zero or non-zero.

Reply:
I've switched to &> in https://github.com/WordPress/openverse/pull/4343/commits/59d705bac0d73642761888966109141f8b395afe